src/Digdir.Domain.Dialogporten.WebApi/Common/Extensions/WebApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,50 @@
+using Azure.Monitor.OpenTelemetry.AspNetCore;
+using OpenTelemetry.Trace;
+using Npgsql;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+
+namespace Digdir.Domain.Dialogporten.WebApi.Common.Extensions;
+
+internal static class WebApplicationBuilderExtensions
+{
+    public static WebApplicationBuilder ConfigureTelemetry(this WebApplicationBuilder builder)
```

Comment:
This is kiiiiing! Dette er kanskje noe vi vil refaktorere ut i det nye aspnet-biblioteket vårt etterhvert?

Reply:
Det er en god ide, la oss se hvordan ting fungerer med OT først